python/cuml/cuml/preprocessing/LabelEncoder.py

```diff
@@ -258,7 +257,7 @@ def fit_transform(self, y, z=None) -> cudf.Series:
 
         return y.cat.codes
 
-    def inverse_transform(self, y: cudf.Series) -> cudf.Series:
+    def inverse_transform(self, y: "cudf.Series"):
```

Comment:
Using strings or `from __future__ import annotations`  for is enough to avoid an import-time attribute access for argument type hints... but not for return types. See https://github.com/rapidsai/cuml/pull/6400#discussion_r1987716383